src/test/regress/expected/multi_modifying_xacts.out

```diff
@@ -119,8 +119,7 @@ EXCEPTION
         RAISE NOTICE 'caught not_null_violation';
 END $$;
 NOTICE:  caught not_null_violation
-COMMIT;
-ERROR:  cannot ROLLBACK TO SAVEPOINT in transactions which modify distributed tables
+ROLLBACK;
 \set VERBOSITY default
```

Comment:
This is exercising the implicit savepoints created by the PL/pgSQL's `BEGIN`/`EXCEPTION`/`END` syntax, I think. The test above tests an explicit `SAVEPOINT` and this tests an implicit one.